.github/workflows/backend-e2e-test.yml

```diff
@@ -52,3 +52,8 @@ jobs:
       - name: run test for plugin skywalking
         working-directory: ./api/test/e2e/skywalking
         run: go test -v
+
+      - name: output test coverage
+        working-directory: ./api/test/testdata
+        run: |
+          go tool cover -func=./integrationcover.out
```

Comment:
@ShiningRush  I think we could upload the result to `Codecov`,  It's Easier to view.

Reply:
Is here any demo?

Reply:
how about this ?
https://github.com/apache/apisix-dashboard/blob/master/.github/workflows/backend-unit-test.yml#L46

Reply:
@nic-chen you can help @ShiningRush to finish this PR

Reply:
OK